### PR TITLE
Added option to disable cursor in screenshots.

### PR DESCRIPTION
**Pull request by Druanae**
Instead of using a flag I've opted to add it as a config option, complete with entry in the settings editor functions in the script. While I'd prefer to have the cursor off myself, I understand that most would prefer to have the cursor visible and so have made it visible by default.
I also fixed some (a lot) of the indentation in the code.

### CODE REVIEW

File: script.sh [hunk 1]
```diff
@@ -161,14 +171,16 @@ function misc() {
...
 	read -p "Please enter your selection: " selection
 	case "$selection" in
 		1 ) notif_prefs;;
 		2 ) scrsave_prefs;;
-		3 ) xclip_prefs;;
-		4 ) settings;;
+		3 ) cursor_prefs;;
+		4 ) xclip_prefs;;
+		5 ) settings;;
 		* ) echo "Invalid selection. Please choose 1, 2 or 3."; misc;;
```
Comment (TTtie) [inline script.sh]: please add 4 and 5 to the selection

File: script.sh [hunk 2]
```diff
@@ -161,14 +171,16 @@ function misc() {
 	echo "Miscellaneous Settings"
 	echo "1) Notification preferences"
 	echo "2) Screenshot saving preferences"
-	echo "3) Clipboard copying preferences"
-	echo "4) Go back"
+	echo "2) Screenshot cursor preferences"
```
Comment (TTtie) [inline script.sh]: there should be 3 instead of 2

**Review by TTtie — CHANGES_REQUESTED** [review]

**Comment by Druanae** [comment]
Done. I didn't catch that, I'll be sure to check over my changes more thoroughly next time.

**Comment by islacantwin** [comment]
yeah, sure, will merge

**Pull request merged**